listener/avalon_listener/tcs_workorder_receipt_handler.py

```diff
@@ -17,7 +17,6 @@
 import base64
 from utility.hex_utils import is_valid_hex_str
 import avalon_crypto_utils.crypto_utility as crypto_utility
-import avalon_crypto_utils.crypto.crypto as crypto
 from error_code.error_status import ReceiptCreateStatus, SignatureStatus,\
```

Comment:
Need to remove openSSL and common cpp build from listener Dockerfile -> https://github.com/hyperledger/avalon/blob/master/listener/Dockerfile#L99

Reply:
Done